gallery/src/components/NxButton/NxButtonIconOnlyExample.tsx

```diff
@@ -5,15 +5,16 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import {faExclamationTriangle, faEdit, faSave} from '@fortawesome/free-solid-svg-icons';
+import {faExclamationTriangle, faEdit, faSave, faTrashAlt } from '@fortawesome/free-solid-svg-icons';
 
 import { NxButton, NxFontAwesomeIcon, NxButtonBar } from '@sonatype/react-shared-components';
 
 const NxButtonIconOnlyExample = () =>
   <NxButtonBar>
-    <NxButton title="Alert" variant="icon-only"><NxFontAwesomeIcon icon={faExclamationTriangle}/></NxButton>
+    <NxButton variant="icon-only" title="Alert"><NxFontAwesomeIcon icon={faExclamationTriangle}/></NxButton>
     <NxButton variant="icon-only" title="Edit"><NxFontAwesomeIcon icon={faEdit}/></NxButton>
     <NxButton variant="icon-only" title="Save"><NxFontAwesomeIcon icon={faSave}/></NxButton>
+    <NxButton variant="icon-only" title="Delete" disabled><NxFontAwesomeIcon icon={faTrashAlt}/></NxButton>
```

Comment:
The tooltip for this button is getting stuck open.  Additionally I don't think disabled buttons necessarily need tooltips since they can't be focused.  Maybe just an `aria-label` instead.

Reply:
> Totally missed that, updated it here: c9b7446

Really I think there are multiple cases that need to be shown here.  We do want the attribute to work, and we want to demonstrate it.  It's also the case that attribute-disabled icon-only buttons don't require `title`s, so maybe we should show an attr-disabled button with no title.  But then it's also the case that the caller might _want_ a tooltip on the button, so we should perhaps additionally show the the way to do that is with the disabled classname.